src/compiler/crystal/syntax/parser.cr

```diff
@@ -3687,10 +3688,12 @@ module Crystal
       node
     end
 
-    def preserve_stop_on_do(new_value = false)
+    def preserve_stop_on_do(new_value = false, call_args? = true)
```

Comment:
Could you use `is_call_args` instead? `call_args?` as a variable name should be a parse error (at least it's not allowed in Ruby, and in Crystal it looks weird). If we don't change it now we'll have to change it later when we fix the parser for this, but I don't want to encourage this names.